hermes-api/build.gradle

```diff
@@ -9,6 +9,10 @@ dependencies {
     compile group: 'com.fasterxml.jackson.jaxrs', name: 'jackson-jaxrs-json-provider', version: versions.jackson
     compile group: 'com.google.guava', name: 'guava', version: versions.guava
     compile group: 'com.damnhandy', name: 'handy-uri-templates', version: '2.0.2'
+    compile group: 'javax.xml.bind', name: 'jaxb-api', version: '2.2.11'
```

Comment:
Do we need all of these dependencies?

Reply:
Those are java ee dependencies removed from jdk 11 and they are need in this package